api/schemas/serializers.py

```diff
@@ -33,18 +33,13 @@ class RegistrationSchemaFormBlockSerializer(JSONAPISerializer):
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
-    page = ser.CharField(max_length=255, read_only=True)
-    section = ser.CharField(max_length=255, read_only=True)
-    help_text = ser.CharField(read_only=True, allow_blank=True)
-    block_id = ser.CharField(max_length=255)
+    answer_id = ser.CharField(max_length=255)
```

Comment:
Only one `-input` block allowed for each chunk_id.  This `-input` block has an `answer_id` that directly maps to the user's answer.